### ISSUE TITLE
Help!

### ISSUE THREAD

**Issue by linyue92027** [hizhangp/yolo_tensorflow#45]
When I trained the network ,there was a value error -"uknown code 'f' for object of type 'str'" in train.py, and the program stoped in line of 'train_timer.remain(step, self.max_iter))'.  Why?

**Comment by zyyhl** [comment]
I also have the same problem. Do you use gpu training? I use gpu+tensorflow-gpu1.2.

**Comment by marshellhu** [comment]
hi， how to use gpu in this program?which should I change?I change the gpu=1 in config.py but it doesn't work

**Comment by hizhangp** [comment]
Comment line `train_timer.remain(step, self.max_iter))` will help.

**Comment by zyyhl** [comment]
I comment line 90 in train.py, but didn't solve the problem, still have the same problem

**Comment by linyue92027** [comment]
def main():
    parser.add_argument('--iou_threshold', default=0.5, type=float)
    parser.add_argument('--gpu', default='1', type=str）##
    args = parser.parse_args()

When you train the model, use the following command :
python train.py --weights YOLO_small.ckpt --gpu1

**Comment by linyue92027** [comment]
I have solved the problem of "uknown code 'f' for object of type 'str'" in train.py:
log_str = '''{} Epoch: {}, 
                                 Step: {}, 
                                 Learning rate: {},
                                 Loss: {:5.3f}\nSpeed: {:.3f}s/iter,
                                 Load: {:.3f}s/iter, 
                                 Remain: {}'''.format(
                        datetime.datetime.now().strftime('%m-%d %H:%M:%S'),
                        self.data.epoch,
                        int(step),
                        round(self.learning_rate.eval(session=self.sess), 6),
                        loss,
                        train_timer.average_time,
                        load_timer.average_time,
                        train_timer.remain(step, self.max_iter))
